pkg/command/install/install.go

```diff
@@ -322,7 +322,8 @@ func installCertManager(ctx *cmdcontext.RWContext) error {
 // apply into the cluster
 // TODO(xinyu): add the version for risingwave.yaml.
 func installOperator(ctx *cmdcontext.RWContext, version string) error {
-	yamlFile, err := Download(RisingWaveUrl, TemDir+"/operator")
+	url := fmt.Sprintf(RisingWaveUrlTemplate, minimumVersion(version))
```

Comment:
We should warn about the minimum version if the specified version is lower than that. It's not a good choice to implicitly change the version.